tests/test_bonanza.rs

```diff
@@ -0,0 +1,2 @@
+mod mock;
+mod tests;
```

Comment:
Can we call it `testsuite`? Then we do not have this kind of `tests/tests` path.

Reply:
done